tools/niminst/niminst.nim

```diff
@@ -63,6 +63,7 @@ type
     nimArgs: string
     debOpts: TDebOptions
     nimblePkgName: string
+    checksum: string
```

Comment:
This is never set?

Reply:
I think that it is accessed in `install.nimf` and `deinstall.nimf` but I'm not 100% sure that it is the same variable.

Reply:
I renamed it to `nimblePkgChecksum` and I set it in the `parseIniFile` procedure the same way as the `nimblePkgName` field.

Reply:
> I think that it is accessed in install.nimf and deinstall.nimf but I'm not 100% sure that it is the same variable.

Well niminst wouldn't compile if you used an undeclared variable.